pkg/build/api/v1/types.go

```diff
@@ -309,6 +309,10 @@ type DockerBuildStrategy struct {
 
 	// ForcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
+
+	// DockerfilePath is the path of the Dockerfile that will be used to build the Docker image,
+	// relative to the root of the context (contextDir).
+	DockerfilePath string `json:"dockerfilePath,omitempty" description:"path of the Dockerfile to use for building the Docker image, relative to the root of the context"`
```

Comment:
Make it explicit in the description, you mean contextDir. Somebody looking at the docs (see [this](https://github.com/openshift/origin/pull/5976/files#diff-32f44d30462f6b04c7cf597f5f94043f)) might have hard time understanding it. 
My proposal:
"... relative to the contextDir, if set"